### PR TITLE
Update readme of app-mode installation

### PR DESCRIPTION
**Pull request by guangbochen**
**Description:**
set master branch chart to dev and update the readme & app-mode installation doc.

### CODE REVIEW

File: deploy/charts/harvester/README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# Harvester Helm Chart
+# Harvester Helm Chart [Development]
 
 Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes.
 
+Note: the master branch is under active development, please use the `stable` branch if you need a reliable version of the Harvester chart.
```
Comment (gitlawr) [inline deploy/charts/harvester/README.md]: s/reliable/stable/

File: docs/app-mode-installation.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ $ helm install harvester harvester \
 Tip: You can create a testing Kubernetes environment in Rancher using the Digital Ocean cloud provider. For details, see [this section.](#digital-ocean-test-environment)
 
 1. Add the Harvester repo `https://github.com/rancher/harvester` to your Rancher catalogs by clicking **Global > Tools > Catalogs.**
-1. Specify the URL and name. The default branch is master. Set the `Helm version` to be `Helm v3`.
+1. Specify the URL and name. Set the branch to `stable` if u need a stable release version. Set the `Helm version` to be `Helm v3`.
```
Comment (gitlawr) [inline docs/app-mode-installation.md]: s/u/you/

**Comment by guangbochen** [comment]
@gitlawr updated.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**